extensions/Prefixes.md

```diff
@@ -43,6 +43,7 @@ Once a prefix is approved, it should be added to the [glTF Validator's list of k
 | `GOOGLE` | Google, LLC | https://www.google.com/ | [#1123](https://github.com/KhronosGroup/glTF/issues/1123) |
 | `GRIFFEL` | Griffel Studio | https://griffelstudio.com/ | [#1861](https://github.com/KhronosGroup/glTF/issues/1861) |
 | `HEVOLUS` | Hevolus Srl | https://www.hevolus.it/ | [#2183](https://github.com/KhronosGroup/glTF/issues/2183) |
+| `ICOSA` | Icosa Foundation | https://icosa.foundation | [#2208](https://github.com/KhronosGroup/glTF/issues/2208) |
```

Comment:
Out of 59 root URLs found in this table, 47 have a trailing `/`. Should we have a consistent pattern?

Reply:
Yes, please make a new PR either removing trailing `/` from all root URLs or adding it everywhere it's missing.